src/main/java/fi/otavanopisto/kuntaapi/server/integrations/management/ManagementPageEntityUpdater.java

```diff
@@ -254,6 +273,23 @@ private void deletePage(PageIdRemoveRequest event, PageId managementPageId) {
         }
       }
     }
+  }
+
+  private IndexablePage createIndexablePage(OrganizationId organizationId, PageId kuntaApiPageId, String language, String content, String title) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "createIndexablePage" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)